<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CD Kiosk</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	background-color: qradialgradient(spread:pad, cx:0.514778, cy:0.295, radius:0.575, fx:0.51, fy:0.261, stop:0 rgba(255, 6, 0, 255), stop:1 rgba(176, 0, 0, 255));
}
.QLabel {
	color: white;
}

.QPushButton {
	font-size:30pt; 
	background: white;
	color: black;
	border: 1px solid lightgray;
	padding: 10px;
	margin: 4px;
	margin-bottom: 3px;
	margin-right: 3px;
}

.QPushButton:pressed, .QPushButton:checked {
	background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(255, 6, 0, 255), stop:0.600985 rgba(176, 0, 0, 255));
	color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelGlobalTitle">
        <property name="styleSheet">
         <string notr="true">font-size: 30pt;</string>
        </property>
        <property name="text">
         <string>CD Kiosk</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="push1Home">
        <property name="styleSheet">
         <string notr="true">font-size: 12pt;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push2Choose">
        <property name="styleSheet">
         <string notr="true">font-size: 12pt;</string>
        </property>
        <property name="text">
         <string>Choose</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push3Cart">
        <property name="styleSheet">
         <string notr="true">font-size: 12pt;</string>
        </property>
        <property name="text">
         <string>Cart</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push4Burn">
        <property name="styleSheet">
         <string notr="true">font-size: 12pt;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Burn</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="centralStack">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,0,0,0,5,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">font-size:20pt</string>
            </property>
            <property name="text">
             <string>Please select what type of CD you want to order</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushCD">
            <property name="styleSheet">
             <string notr="true">.QPushButton {
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Audio CD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushMP3">
            <property name="styleSheet">
             <string notr="true">.QPushButton {
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>MP3 CD</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushAdministration">
            <property name="styleSheet">
             <string notr="true">border-radius: 10px</string>
            </property>
            <property name="text">
             <string>Administration</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="choose">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size:20pt</string>
          </property>
          <property name="text">
           <string>Select Speeches</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="viewSpeeches">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushAddToCart">
            <property name="styleSheet">
             <string notr="true">border-top-left-radius: 10px;
border-bottom-left-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/gui/icons/add.png</normaloff>:/gui/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushProceedToReview">
            <property name="styleSheet">
             <string notr="true">font-weight: 800;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Proceed</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="review">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>-1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-size: 20pt;</string>
          </property>
          <property name="text">
           <string>Items in cart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listCart">
          <property name="styleSheet">
           <string notr="true">font-size: 20pt;</string>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushBackToChoose">
            <property name="styleSheet">
             <string notr="true">font-weight: 800;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Add more</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/gui/icons/add.png</normaloff>:/gui/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushRemoveFromCart">
            <property name="styleSheet">
             <string notr="true">border-top-right-radius: 10px;
border-bottom-right-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Remove Selected</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/gui/icons/remove.png</normaloff>:/gui/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushProceedToBurn">
            <property name="styleSheet">
             <string notr="true">font-weight: 800;
border-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Burn</string>
            </property>
            <property name="icon">
             <iconset resource="../../res/res.qrc">
              <normaloff>:/gui/icons/burn.png</normaloff>:/gui/icons/burn.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="burn">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="CDDisplayWidget" name="jobVisualisation" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelJobProgress">
            <property name="text">
             <string>Progress: 45 %</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="listjobs">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CDDisplayWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/cddisplaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushProceedToBurn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showBurnPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1208</x>
     <y>779</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBackToChoose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showChoosePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>779</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushProceedToReview</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCartPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1177</x>
     <y>779</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showChoosePage()</slot>
  <slot>showCartPage()</slot>
  <slot>showBurnPage()</slot>
  <slot>updateChooseLists()</slot>
  <slot>updateSpeechList()</slot>
 </slots>
</ui>
